lib/twitch.rb

```diff
@@ -33,7 +33,7 @@ def self.followed_ids(login)
 
     class << self
       def get(login)
-        route(login).get(Twitch.headers)
+        route(login.twitch_id).get(Twitch.headers)
```

Comment:
Okie dokie, I'll change the calls to just pass in the twitch id, and then in the functions rename the parameters to just id.